kernels/openvx-core/host/vx_accumulate_weighted_host.c

```diff
@@ -297,7 +297,7 @@ vx_status tivxAddKernelAccumulateWeighted(vx_context context)
         {
             status = vxAddParameterToKernel(kernel,
                         index,
-                        (vx_enum)VX_OUTPUT,
+                        (vx_enum)VX_BIDIRECTIONAL,
```

Comment:
This change is modifying the existing accumulate square kernel but the spec extension defines new kernels.

Reply:
it should be a bidirectional input if you refer to the Khronos specification vers.1.1.